dbt/include/bigquery/macros/materializations/copy.sql

```diff
@@ -1,6 +1,9 @@
 {% materialization copy, adapter='bigquery' -%}
 
   {# Setup #}
+    -- grab current tables grants config for comparision later on
+  {# TODO: this is a guess I need to circle back to - does the BQ copy API also copy grants automatically?  If yes it seems like we would want to option to overwrite them if needed? #}
```

Comment:
> does the BQ copy API also copy grants automatically?

Great question. I don't know the answer. This materialization is also still undocumented (which is on me: https://github.com/dbt-labs/docs.getdbt.com/pull/826)